docs/functions/data.rst

```diff
@@ -325,6 +328,10 @@ Environment
 
    .. versionadded:: 2.0
 
+   .. versionchanged:: 2.2
+      Can be disabled by :attr:`LP_ENABLE_MUX`,
+      except if ``--internal`` is passed.
```

Comment:
It's fine to mention the `--internal` flag, but it should also be mentioned that it is for internal use only.